[rospy.client][INFO] 2017-10-18 07:32:49,072: init_node, name[/state_driving_away], pid[5248]
[xmlrpc][INFO] 2017-10-18 07:32:49,074: XML-RPC server binding to 0.0.0.0:0
[rospy.init][INFO] 2017-10-18 07:32:49,075: ROS Slave URI: [http://10.0.0.116:34176/]
[xmlrpc][INFO] 2017-10-18 07:32:49,076: Started XML-RPC server [http://10.0.0.116:34176/]
[rospy.impl.masterslave][INFO] 2017-10-18 07:32:49,076: _ready: http://10.0.0.116:34176/
[xmlrpc][INFO] 2017-10-18 07:32:49,084: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2017-10-18 07:32:49,089: Registering with master node http://10.0.0.116:11311
[rospy.init][INFO] 2017-10-18 07:32:49,176: registered with master
[rospy.rosout][INFO] 2017-10-18 07:32:49,177: initializing /rosout core topic
[rospy.rosout][INFO] 2017-10-18 07:32:49,206: connected to core topic /rosout
[rospy.simtime][INFO] 2017-10-18 07:32:49,215: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2017-10-18 07:32:49,387: topic[/state_command] adding connection to [/state_avoiding_obstacle], count 0
[rospy.internal][INFO] 2017-10-18 07:32:49,415: topic[/state_command] adding connection to [http://10.0.0.116:33147/], count 0
[rospy.internal][INFO] 2017-10-18 07:32:49,421: topic[/state_command] adding connection to [http://10.0.0.116:34176/], count 0
[rospy.internal][INFO] 2017-10-18 07:32:49,445: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2017-10-18 07:32:49,455: topic[/state_command] adding connection to [/state_driving_away], count 1
[rospy.internal][INFO] 2017-10-18 07:32:49,899: topic[/state_command] adding connection to [/state_driving_to_cone], count 2
[rospy.internal][INFO] 2017-10-18 07:32:49,907: topic[/state_command] adding connection to [http://10.0.0.116:42801/], count 2
[rospy.internal][INFO] 2017-10-18 07:32:49,982: topic[/state_command] adding connection to [http://10.0.0.116:41071/], count 3
[rospy.internal][INFO] 2017-10-18 07:32:50,003: topic[/state_command] adding connection to [/state_follow_waypoint], count 3
[rospy.internal][INFO] 2017-10-18 07:32:50,690: topic[/mavros/local_position/pose] adding connection to [http://10.0.0.116:42365/], count 0
[rospy.internal][INFO] 2017-10-18 07:32:50,715: topic[/mavros/setpoint_raw/target_local] adding connection to [http://10.0.0.116:42365/], count 0
[rospy.internal][INFO] 2017-10-18 07:32:50,742: topic[/mavros/state] adding connection to [http://10.0.0.116:42365/], count 0
[rospy.internal][INFO] 2017-10-18 07:32:50,775: topic[/mavlink/from] adding connection to [http://10.0.0.116:42365/], count 0
[rospy.internal][INFO] 2017-10-18 07:32:50,777: topic[/mavros/battery] adding connection to [http://10.0.0.116:42365/], count 0
[rospy.internal][INFO] 2017-10-18 07:32:50,971: topic[/mavros/mission/waypoints] adding connection to [http://10.0.0.116:42365/], count 0
[rospy.internal][INFO] 2017-10-18 07:32:51,078: topic[/mavros/rc/override] adding connection to [/mavros], count 0
[rospy.internal][INFO] 2017-10-18 07:32:51,366: topic[/exec_command] adding connection to [/exec], count 0
[rospy.internal][INFO] 2017-10-18 07:32:51,443: topic[/state_command] adding connection to [http://10.0.0.116:38599/], count 4
[rospy.core][INFO] 2017-10-18 07:33:29,892: signal_shutdown [signal-2]
[rospy.internal][INFO] 2017-10-18 07:33:30,079: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2017-10-18 07:33:30,080: topic[/mavros/rc/override] removing connection to /mavros
[rospy.internal][INFO] 2017-10-18 07:33:30,081: topic[/state_command] removing connection to /state_avoiding_obstacle
[rospy.internal][INFO] 2017-10-18 07:33:30,082: topic[/state_command] removing connection to /state_driving_away
[rospy.internal][INFO] 2017-10-18 07:33:30,083: topic[/state_command] removing connection to /state_driving_to_cone
[rospy.internal][INFO] 2017-10-18 07:33:30,084: topic[/state_command] removing connection to /state_follow_waypoint
[rospy.internal][INFO] 2017-10-18 07:33:30,086: topic[/exec_command] removing connection to /exec
[rospy.internal][INFO] 2017-10-18 07:33:30,088: topic[/mavros/mission/waypoints] removing connection to http://10.0.0.116:42365/
[rospy.internal][ERROR] 2017-10-18 07:33:30,090: Unable to initiate TCP/IP socket to 10.0.0.116:36384 (http://10.0.0.116:34176/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2017-10-18 07:33:30,106: Unable to initiate TCP/IP socket to 10.0.0.116:38431 (http://10.0.0.116:33147/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.topics][ERROR] 2017-10-18 07:33:30,113: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 838, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.internal][INFO] 2017-10-18 07:33:30,114: topic[/mavros/state] removing connection to http://10.0.0.116:42365/
[rospy.internal][INFO] 2017-10-18 07:33:30,116: topic[/mavlink/from] removing connection to http://10.0.0.116:42365/
[rospy.internal][ERROR] 2017-10-18 07:33:30,121: Unable to initiate TCP/IP socket to 10.0.0.116:38642 (http://10.0.0.116:42801/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2017-10-18 07:33:30,122: topic[/mavros/local_position/pose] removing connection to http://10.0.0.116:42365/
[rospy.internal][INFO] 2017-10-18 07:33:30,124: topic[/mavros/battery] removing connection to http://10.0.0.116:42365/
[rospy.internal][INFO] 2017-10-18 07:33:30,125: topic[/state_command] removing connection to http://10.0.0.116:33147/
[rospy.internal][INFO] 2017-10-18 07:33:30,126: topic[/state_command] removing connection to http://10.0.0.116:34176/
[rospy.internal][INFO] 2017-10-18 07:33:30,126: topic[/state_command] removing connection to http://10.0.0.116:42801/
[rospy.internal][INFO] 2017-10-18 07:33:30,128: topic[/state_command] removing connection to http://10.0.0.116:41071/
[rospy.internal][ERROR] 2017-10-18 07:33:30,129: Unable to initiate TCP/IP socket to 10.0.0.116:38022 (http://10.0.0.116:41071/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2017-10-18 07:33:30,139: Unable to initiate TCP/IP socket to 10.0.0.116:44772 (http://10.0.0.116:38599/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2017-10-18 07:33:30,140: topic[/state_command] removing connection to http://10.0.0.116:38599/
[rospy.topics][ERROR] 2017-10-18 07:33:30,141: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 838, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.internal][INFO] 2017-10-18 07:33:30,152: topic[/mavros/setpoint_raw/target_local] removing connection to http://10.0.0.116:42365/
[rospy.impl.masterslave][INFO] 2017-10-18 07:33:30,155: signal-2
[rospy.core][INFO] 2017-10-18 07:33:30,266: signal_shutdown [atexit]
